src/clj/ataru/hakija/hakija_routes.clj

```diff
@@ -17,15 +26,16 @@
                :tags [{:name "application-api" :description "Application handling"}]}}
     (api/context "/api" []
                  :tags ["application-api"]
-                 (api/GET "/form/:id" [id]
-                          (ok
-                            {:form {:x id}})))))
+                 (api/GET "/form/:id" []
+                          :path-params [id :- Long]
+                          :return s/Any
+                          (ok (fetch-form id))))))
```

Comment:
`fetch-form` seems to return a `ok` or `not-found` response, so is the wrapping needed here?